charts/node/values.yaml

```diff
@@ -54,58 +89,102 @@ ingress:
-  #chainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
-  #chainDataSnapshotFormat: lz4
-  #chainPath: ""
-  #chainDataKubernetesVolumeSnapshot: ""
-  #chainDataGcsBucketUrl: ""
+
+  ## Configuration to download collator snapshot from remote URL
+  ## 
+  # chainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
+  # chainDataSnapshotFormat: lz4
+  # chainDataKubernetesVolumeSnapshot: ""
+  # chainDataGcsBucketUrl: ""
+  # chainPath: ""
+
+  ## Deploy a collator node
+  ## ref: https://wiki.polkadot.network/docs/learn-collator
+  ## If Collator is enabled, collator image must be used
+  ##
   collator:
     isParachain: false
+
+    ## Name of the Relay Chain to connect
+    ##
     relayChain: polkadot
-  #  relayChainCustomChainspecUrl: ""
-  relayChainCustomChainspecPath: "/data/relay_chain_chainspec.json"
-  #  relayChainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
-  #  relayChainDataSnapshotFormat: lz4
-  #  relayChainPath: ""
-  #  relayChainDataKubernetesVolumeSnapshot: ""
-  #  relayChainDataGcsBucketUrl: ""
-  #  relayChainFlags:
-  enableStartupProbe: true
-  enableReadinessProbe: true
+
+    ## Expose metrics via Prometheus format in /metrics endpoint
+    ## Pass the following args:
+    ##   - "--prometheus-external" \
+    ##   - "--prometheus-port {{ .Values.node.collator.prometheus.port }}"
+    ##
+    prometheus:
+      enabled: true
+      port: 9625
+  
+    ## File containing chainspec of relaychain for
+    ## 
+    relayChainCustomChainspecPath: "/data/relay_chain_chainspec.json"
+    # relayChainCustomChainspecUrl: ""
+
+    ## Configuration to download relaychain snapshot from remote URL
+    ## 
+    # relayChainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
+    # relayChainDataSnapshotFormat: lz4
+    # relayChainDataKubernetesVolumeSnapshot: ""
+    # relayChainDataGcsBucketUrl: ""
+    # relayChainPath: ""
+
+    ## Additional args to use for relay chain node
+    ##
+    # relayChainFlags: []
+
+  ## Additional args to use
+  ## 
   flags:
     - "--rpc-external"
     - "--ws-external"
     - "--rpc-methods=safe"
     - "--rpc-cors=all"
-    - "--prometheus-external"
+  
+  ## Keys to use by the node
+  ## ref: https://wiki.polkadot.network/docs/learn-keys
+  ##
   keys: {}
-  #  - type: "gran"
-  #    scheme: "ed25519"
-  #    seed: "//Alice//gran"
-  #  - type: "babe"
-  #    scheme: "sr25519"
-  #    seed: "//Alice//babe"
+  # - type: "gran"
+  #   scheme: "ed25519"
+  #   seed: "//Alice//gran"
+  # - type: "babe"
+  #   scheme: "sr25519"
+  #   seed: "//Alice//babe"
+
+  ## Component to inject secrets via annotation of Hashicorp Vault
+  ## ref: https://www.vaultproject.io/docs/platform/k8s/injector/annotations
+  ##
   vault:
+    ## Vault annotations to use
+    ##
     #authPath: ""
     #authRole: ""
     #authType: ""
     #authConfigType: ""
     #authConfigServiceAccount: ""
+
+    ## Keys to use by the node via vault
```

Comment:
`Keys to use by the node via vault`, this phrasing is hard to get right but it could be something like: `Keys to fetch from Vault and set on the node`. But it's tangential to the current PR.